lib/tasks/bulkrax_tasks.rake

```diff
@@ -5,26 +5,27 @@ namespace :bulkrax do
   task rerun_all_exporters: :environment do
     if defined?(::Hyku)
       Account.find_each do |account|
-        puts "=============== updating #{account.name} ============"
         next if account.name == "search"
         switch!(account)
+        puts "=============== updating #{account.name} ============"
 
-        rerun_exporters_and_delete_zips
+        make_new_exports
 
         puts "=============== finished updating #{account.name} ============"
       end
     else
-      rerun_exporters_and_delete_zips
+      make_new_exports
     end
   end
 
-  def rerun_exporters_and_delete_zips
+  def make_new_exports
+    # delete the existing folders and zip files
```

Comment:
deletes everything in "tmp/exports" and creates new paths.

left it in here even for hyrax apps because if the app is upgrading to 4.0+, it will still need the new file structure outlined in #582.